docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -393,12 +421,19 @@ Start by adding the following dependencies to your test project:
 testImplementation("io.rest-assured:rest-assured")
 testImplementation("io.quarkus:quarkus-junit5")
 ----
+====
 
-[[integration-testing-wiremock]]
-==== Wiremock
+//@sberyozkin - It might be good to add a stament here about the different mechanisms or methods to integration testing. For example, WireMock... etc
```

Comment:
Sure, may be just underneath `Testing section` above - but I can't promise I'll remember this comment :-), may be worth opening an issue ? Or I can push an update to your PR ?

Reply:
@sberyozkin - :+1: +1 - An update to my PR would be perfect. Thank you.

Reply:
Sounds good @michelle-purcell